cmd/clusterctl/client/init.go

```diff
@@ -73,6 +73,9 @@ type InitOptions struct {
 	// IgnoreValidationErrors allows for skipping the validation of provider installs.
 	// NOTE this should only be used for development
 	IgnoreValidationErrors bool
+
+	// To ensure the provider CRD is installed
+	ProviderCRDInstalled bool
```

Comment:
```suggestion
	allowMissingProviderCRD bool
```

Please also adjust the usages of this variable accordingly.